### PR TITLE
ER-1410 blocked transferred vacancies employer

### PR DESCRIPTION
**Pull request by chrisjensenuk**
Employer.Web alert for vacancies that have been transferred because a Provider has been blocked by QA.

### CODE REVIEW

File: src/Shared/Recruit.Vacancies.Client/Domain/Entities/AlertType.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
     public enum AlertType
     {
         TransferredVacancies,
-        BlockedProvider
+        BlockedProvider,
```
Comment (dev4valtech) [inline src/Shared/Recruit.Vacancies.Client/Domain/Entities/AlertType.cs]: The names could be improved to show the effect and reason for example:  VacanciesClosedAsProviderBlocked, VacanciesTransferredAsProviderBlocked

File: src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml [hunk 2]
```diff
@@ -22,10 +22,14 @@
...
+    @if (Model.ShowEmployerRevokedTransferredVacanciesAlert)
+    {
+        <partial name="_EmployerRevokedTransferredVacanciesAlert" for="EmployerRevokedTransferredVacanciesAlert" />
+    }
 
-    @if (Model.ShowTransferredVacanciesAlert)
+    @if (Model.ShowBlockedProviderTransferredVacanciesAlert)
     {
-        <partial name="_TransferredVacanciesAlert" for="TransferredVacanciesAlert" />
+        <partial name="_BlockedProviderTransferredVacanciesAlert" for="BlockedProviderTransferredVacanciesAlert" />
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: Can you use the PartialNames constants file for referencing partial view names please?

Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: 👍

File: src/Shared/Recruit.Vacancies.Client/Domain/Entities/AlertType.cs (hunk 1 above)
Comment (chrisjensenuk) [reply src/Shared/Recruit.Vacancies.Client/Domain/Entities/AlertType.cs]: 👍 I'll review this at the end.

**Comment by chrisjensenuk** [comment]
Changes are in #697

**Pull request closed without merging**